lms/static/sass/course/modules/_calculator.scss

```diff
@@ -13,10 +15,11 @@ div.calc-main {
 
   a.calc {
     @include text-hide();
-    background: url("../images/calc-icon.png") rgba(#111, .9) no-repeat center;
+    @include transition(background-color $tmg-f2 ease-in-out 0s);
+    background: url("../images/calc-icon.png") $black-t3 no-repeat center;
```

Comment:
Also be sure to remove the .png file from the repo if we switch over to font awesome on this one.